### PR TITLE
MODOAIPMH-76  Release 1.1.1

### PR DESCRIPTION
**Pull request by javalkar-varun**
**Purpose:**
https://issues.folio.org/browse/MODOAIPMH-76 - release 1.1.1

**Approach:**
Update NEWS.md
Update SNAPSHOT version in pom.xml

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-oai-pmh</artifactId>
-  <version>1.2.0-SNAPSHOT</version>
```
Comment (khandramai) [inline pom.xml]: No need to make this now. Version changing will happen during release plugin execution.

File: NEWS.md [hunk 2]
```diff
@@ -1,4 +1,14 @@
-## 1.1.1 (UnReleased)
+## 1.1.2 (UnReleased)
+
+## 1.1.1 (Released)
```
Comment (khandramai) [inline NEWS.md]: Please, be careful with versioning. You can use semantic versioning guide https://semver.org/spec/v2.0.0.html. So here I don't see reasons for patch number increasing. It seems minor version should be increased, i.e. 1.2.0.

**Comment by khandramai** [comment]
JIC, release procedure described here https://wiki.folio.org/display/FOLIJET/Acquisitions+-+Release+Procedures.

**Comment by javalkar-varun** [comment]
Thanks for your comments @khandramai. Since we are not releasing this sprint, I'll close this PR and we can revisit when the time comes.

**Pull request closed without merging**